samples/CallWithChat/src/app/views/HomeScreen.tsx

```diff
@@ -46,7 +55,15 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
   const [chosenCallOption, setChosenCallOption] = useState<IChoiceGroupOption>(callOptions[0]);
   const [teamsLink, setTeamsLink] = useState<TeamsMeetingLinkLocator>();
 
+  /* @conditional-compile-remove(PSTN-calls) */
+  const [alternateCallerId, setAlternateCallerId] = useState<string>();
+  /* @conditional-compile-remove(PSTN-calls) */
+  const [outboundParticipants, setOutboundParticipants] = useState<string>();
+
   const teamsCallChosen: boolean = chosenCallOption.key === 'TeamsMeeting';
+
+  /* @conditional-compile-remove(PSTN-calls) */
+  const outBoundCallChosen: boolean = chosenCallOption.key === 'outBoundCall';
   const buttonEnabled = displayName && (!teamsCallChosen || teamsLink);
```

Comment:
Should there be an extra buttonEnabled check here to ensure a call cannot be started if `outBoundCall` is chosen but no `alternateCallerId` and `outboundParticipants` supplied

Reply:
good call will update this!

Reply:
I am going to look at this in the dialpad PR: https://github.com/Azure/communication-ui-library/pull/2114 since the mode of starting the call is split up there.